src/site/content/en/blog/introducing-tooling-report/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Choose the best build tool for your project with tooling.report
+subhead: Select and configure build tools based on best practices.
+description: |
+  Select and configure build tools based on best practices.
+scheduled: true
+date: 2020-06-29
+hero: hero.jpg
```

Comment:
Something like this might be nice for hero:

![Screen Shot 2020-06-25 at 9 00 08 AM](https://user-images.githubusercontent.com/4713486/85756087-5c33d480-b6c3-11ea-8f4e-9a653566288e.jpg)

Dimensions should be 3200 by 960

We'll need to provide a thumbnail, which should be 896 by 480

Reply:
I'll leave it up to someone else to push up a hero image to this branch

Reply:
@argyleink can you whip something up? We need this branch to land by Monday.

Reply:
sure, hero and thumbnail incoming 👍

Reply:
I added hero image and added thumbnail I had.

Reply:
Oh brilliant! Thank you, Mariko!

Reply:
I'll wait for Adam's (arguably bette) graphics tho!

Reply:
your thumb was rad! i did add a hero header tho!

Reply:
![Screen Shot 2020-06-25 at 10 20 15 AM](https://user-images.githubusercontent.com/1134620/85768790-3e1fa180-b6ce-11ea-86e4-21f1831bb954.png)

Reply:
nice

Reply:
nice